himbaechel/himbaechel_api.h

```diff
@@ -102,6 +109,20 @@ struct HimbaechelAPI
     virtual bool isClusterStrict(const CellInfo *cell) const;
     virtual bool getClusterPlacement(ClusterId cluster, BelId root_bel,
                                      std::vector<std::pair<CellInfo *, BelId>> &placement) const;
+
+    // Graphics
+    virtual uint32_t gfxAttributes() { return 0; }
```

Comment:
I really don't like this function, can you think of a better way here, like perhaps a struct that the architecture configures?